buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidTarget.groovy

```diff
@@ -125,7 +124,7 @@ abstract class AndroidTarget extends JavaLibTarget {
         }.flatten() as Set<String>
     }
 
-    protected String extractMergedManifest() {
+    public AndroidTarget extractMergedManifest() {
```

Comment:
This does not make sense to me. Why is this returning this object instead of a string?